qdrant-landing/content/blog/qdrant-unstructured.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Loading Unstructured Data into Qdrant from the Terminal
+slug: qdrant-unstructured
+short_description: Loading Unstructured Data into Qdrant from the Terminal
+description: Learn how to simplify the process of loading unstructured data into Qdrant using the Qdrant Unstructured destination.
+date: 2024-01-09T00:41:38+05:30
+author: Anush Shetty
+tags:
+  - integrations
+  - qdrant
+  - unstructured
+---
+
+Building powerful applications with Qdrant starts with loading vector representations into the system. Traditionally, this involves scraping or extracting data from sources, performing operations such as cleaning, chunking, and generating embeddings, and finally loading it into Qdrant. This process can be complex, but the Qdrant ingestion destination in Unstructured aims to simplify it.
+
+In this blog post, we'll demonstrate how to load data into Qdrant from the channels of a Discord server. The process is similar for the [20+ vetted data sources](https://unstructured-io.github.io/unstructured/ingest/source_connectors.html) supported by Unstructured.
+
+### Prerequisites
+
+- A running Qdrant instance. Refer to our [Quickstart guide](https://qdrant.tech/documentation/quick-start/) to set up an instance.
+- A Discord bot token. Generate one [here](https://discord.com/developers/applications) after adding the bot to your server.
+- Unstructured CLI with the required extras. Install it with the following command:
+
+```bash
+pip install unstructured[discord,local-inference,qdrant]
+```
+
+Once you have the prerequisites in place, let's begin the data ingestion.
+
+### Retrieving Data from Discord
+
+To generate structured data from Discord using the Unstructured CLI, run the following command:
```

Comment:
I think we can link to https://www.pythondiscord.com/pages/guides/pydis-guides/contributing/obtaining-discord-ids/.
Since the official Discord docs don't specify this.

Reply:
https://github.com/qdrant/landing_page/pull/512/commits/7ed375b935c4a9d283009ad4ace621f278839bec